docs/compute.md

```diff
@@ -344,7 +344,24 @@ Other useful commands for listing the VM core affinity and cores per NUMA node(s
 ```
 xl vcpu-list
 xenpm get-cpu-topology
+xl info --numa
 ```
+
+Other ways control the vCPUs placement, using the `xl` CLI:
+```
+xl vcpu-pin <Domain> <vcpu id> <cpu id>
+xl vcpu-pin Domain\ 0 all 2-5
+```
+
+Or using the CPUPool functionality:
+```
+xl cpupool-numa-split # Will create a cpupool by NUMA node
+xl cpupool-migrate <VM> <Pool> # Will migrate a VM to the given pool
+```
+
+:warning: Be careful, the changes done using `xl` only affect vCPU at the moment, the memory of the VM will not be moved between node nor the pinning stay after a reboot. You need to use `xe` for it to be taken into account at the VM startup. :warning:
```

Comment:
You can use the Warning block from vuepress: https://v1.vuepress.vuejs.org/guide/markdown.html#custom-containers

Reply:
It's what I was trying to do :grimacing: